15-draft.json

```
@@ -0,0 +1,1187 @@
+{
+  "$id": "https://schema.spaceapi.io/15-draft.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "SpaceAPI 15",
```

Same here. And maybe add a `(Draft)`?

```suggestion
  "description": "SpaceAPI v15 (Draft)",
```